src/dropdown.ts

```diff
@@ -587,30 +591,18 @@ export default class Dropdown extends LitElement implements FormControl {
 
   formResetCallback(): void {
     for (const option of this.#optionElements) {
-      const isInitiallySelected = option.hasAttribute('selected');
+      // `#onOptionsSelectedChange()` is called when an option is selected. It updates
+      // `this.selectedAndEnabledOptions`. Deselecting every option before reselecting
+      // those in `value` ensures tags appear in the same order they were initially.
+      option.selected = false;
 
-      if (!isInitiallySelected) {
-        option.selected = false;
+      const isInitiallySelected =
+        option.hasAttribute('selected') && !option.disabled;
```

Comment:
The `!option.disabled` condition is a bug fix, technically. But form resets were broken in a [larger way](https://github.com/CrowdStrike/glide-core/pull/932/files#diff-578302523fd9647f4ff6769a88c274dcf582deac425b2e9a3127f2215b0d8efcR8). So I omitted a changeset.